app/src/hooks/useGraphMarketsFromQuestion.tsx

```diff
@@ -50,13 +51,10 @@ export const useGraphMarketsFromQuestion = (questionId: string): Result => {
     if (!questionId) setMarkets([])
   }, [questionId])
 
-  if (data && data.question && data.question.indexedFixedProductMarketMakers.length > 0 && markets.length === 0) {
```

Comment:
Don't we want to leave this check: `data.question.indexedFixedProductMarketMakers.length > 0` so we can error if no markets are returned?

Reply:
@KadenZipfel , if no markets are returned, indexedFixedProductMarketMakers will be an empty array and isObjectEqual will return true.
Don't need to check `data.question.indexedFixedProductMarketMakers.length > 0`